handler.go

```diff
@@ -115,6 +118,101 @@ type TorrentManager struct {
 	Updates time.Duration
 
 	hotCache *lru.Cache
+
+	// For manage torrents Seeding by SeedingLocal(), true/false means seeding/pause
+	localSeedLock  sync.RWMutex
+	localSeedFiles map[string]bool
+}
+
+// can only call by fs.go: 'SeedingLocal()'
+func (tm *TorrentManager) addLocalSeedFile(ih string) bool {
+	if !common.IsHexAddress(ih) {
+		return false
+	}
+	ih = strings.TrimPrefix(strings.ToLower(ih), common.Prefix)
+
+	tm.localSeedLock.Lock()
+	defer tm.localSeedLock.Unlock()
+
+	if _, ok := GoodFiles[ih]; ok {
+		return false
+	}
+	tm.localSeedFiles[ih] = true
+	return true
+}
+
+// only files in map:localSeedFile can be paused!
+func (tm *TorrentManager) pauseLocalSeedFile(ih string) error {
+	if !common.IsHexAddress(ih) {
+		return errors.New("invalid infohash format")
+	}
+	ih = strings.TrimPrefix(strings.ToLower(ih), common.Prefix)
+
+	tm.localSeedLock.Lock()
+	defer tm.localSeedLock.Unlock()
+	if valid, ok := tm.localSeedFiles[ih]; !ok {
+		return errors.New(fmt.Sprintf("Not Local Seeding File<%s>", ih))
+	} else if _, ok := GoodFiles[ih]; ok {
+		return errors.New(fmt.Sprintf("Cannot Pause On-Chain GoodFile<%s>", ih))
+	} else if !valid {
+		return errors.New(fmt.Sprintf("Local Seeding File Is Not Seeding<%s>", ih))
+	}
+
+	if t := tm.getTorrent(ih); t != nil {
+		log.Debug("TorrentFS", "from seed to pause", "ok")
+		t.Pause()
+	}
+	tm.localSeedFiles[ih] = false
+
```

Comment:
```tm.localSeedFiles[ih] = !t.Paused()``` I think the torrent status will be paused certainly , but get the torrent current status  for checking out will be more graceful.